test/github.spec.ts

```diff
@@ -57,6 +57,36 @@ describe("Github", async() => {
   after(async() => {
     await mockAgent.close();
   });
+  test("should throw an error if the GITHUB_TOKEN environment variable is not defined", async() => {
+    const token = process.env.GITHUB_TOKEN;
+
+    process.env.GITHUB_TOKEN = "";
+
+    try {
+      assert.rejects(
+        async() => new Github("NodeSecure"),
```

Comment:
Actually Github constructor is:
```js
constructor(orgName: string) {
  this.orgName = orgName;
}
```

You'll need to update the code if you want it to throws if there is no `GITHUB_TOKEN` env.

Use `assert.throws` instead of `assert.rejects`, `rejects` is for promises, a constructor cannot be async.

You can also change the message: `Unauthorized` -> `Missing GITHUB_TOKEN env` (something like this)

And you don't need to re-set the `GITHUB_TOKEN` env after the test, you can remove `try-finally`